### PR TITLE
Add anti-affinity for databases and their replicas

### PR DESCRIPTION
**Pull request by jakemalachowski**
Update the operator to set the `podAntiAffinity` to the pgcluster name. 

Results in the following for both the primary and the replica:
```
    podAntiAffinity:
      requiredDuringSchedulingIgnoredDuringExecution:
      - labelSelector:
          matchExpressions:
          - key: pg-cluster
            operator: In
            values:
            - dpg-cbdcjqa87d5ng7pkk79g-a
        topologyKey: kubernetes.io/hostname
```

### CODE REVIEW

File: chart/postgres-operator/files/postgres-operator/cluster/1/affinity.json [hunk 1]
```diff
@@ -8,10 +8,23 @@
...
-                }   
-            }]  
-            }   
+                    ]
+                }]
+                }
+            }]
+            },
+            "podAntiAffinity": {
+            "requiredDuringSchedulingIgnoredDuringExecution": [{
```
Comment (maherbeg) [inline chart/postgres-operator/files/postgres-operator/cluster/1/affinity.json:17]: Nit: does this need to be indented one tab over?

**Review by maherbeg — APPROVED** [review]
nice!

File: chart/postgres-operator/files/postgres-operator/cluster/1/affinity.json (hunk 1 above)
Comment (jakemalachowski) [reply chart/postgres-operator/files/postgres-operator/cluster/1/affinity.json:17]: I was actually just copying the formatting of the existing config above

**Review by hieurender — APPROVED** [review]

**Comment by jakemalachowski** [comment]
When testing a separate change, I noticed that our code copies the configmap from the default namespace when create the operator rather than using the CMs defined here. That configmap is managed by [clusterops](https://github.com/renderinc/api/blob/master/clusterops/base/base/postgres-operator-conf.yaml). I'll open a PR there instead.

**Pull request closed without merging**